[2025-03-27T10:27:56.804+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_aqi_to_postgres.fetch_and_insert_aqi scheduled__2025-03-27T09:00:00+00:00 [queued]>
[2025-03-27T10:27:56.811+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_aqi_to_postgres.fetch_and_insert_aqi scheduled__2025-03-27T09:00:00+00:00 [queued]>
[2025-03-27T10:27:56.811+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-27T10:27:56.824+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_and_insert_aqi> on 2025-03-27 09:00:00+00:00
[2025-03-27T10:27:56.827+0000] {standard_task_runner.py:60} INFO - Started process 1882 to run task
[2025-03-27T10:27:56.829+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_aqi_to_postgres', 'fetch_and_insert_aqi', 'scheduled__2025-03-27T09:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/insert_aqi_to_db.py', '--cfg-path', '/tmp/tmpfozz6064']
[2025-03-27T10:27:56.831+0000] {standard_task_runner.py:88} INFO - Job 28: Subtask fetch_and_insert_aqi
[2025-03-27T10:27:56.840+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-27T10:27:56.861+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_aqi_to_postgres.fetch_and_insert_aqi scheduled__2025-03-27T09:00:00+00:00 [running]> on host 79a5714298e2
[2025-03-27T10:27:56.904+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='insert_aqi_to_postgres' AIRFLOW_CTX_TASK_ID='fetch_and_insert_aqi' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-27T09:00:00+00:00'
[2025-03-27T10:27:57.200+0000] {insert_aqi_to_db.py:37} INFO - âœ… AQI in Bangkok: 117 at 2025-03-27 10:27:57.200327
[2025-03-27T10:27:57.205+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-27T10:27:57.207+0000] {sql.py:450} INFO - Running statement: 
                INSERT INTO air_quality_data (city, aqi, timestamp_utc)
                VALUES (%s, %s, %s);
            , parameters: ('Bangkok', 117, datetime.datetime(2025, 3, 27, 10, 27, 57, 200327))
[2025-03-27T10:27:57.208+0000] {sql.py:459} INFO - Rows affected: 1
[2025-03-27T10:27:57.210+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-03-27T10:27:57.216+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_aqi_to_postgres, task_id=fetch_and_insert_aqi, execution_date=20250327T090000, start_date=20250327T102756, end_date=20250327T102757
[2025-03-27T10:27:57.263+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-27T10:27:57.270+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
